kafka-ui-react-app/src/components/Brokers/Broker/__test__/Broker.spec.tsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import { render, WithRoute } from 'lib/testHelpers';
+import { screen, waitFor } from '@testing-library/dom';
+import { clusterBrokerPath } from 'lib/paths';
+import fetchMock from 'fetch-mock';
+import { clusterStatsPayloadBroker } from 'redux/reducers/brokers/__test__/fixtures';
+import { act } from '@testing-library/react';
+import Broker from 'components/Brokers/Broker/Broker';
+import { BrokersLogdirs } from 'generated-sources';
+
+describe('Broker Component', () => {
+  afterEach(() => fetchMock.reset());
+
+  const clusterName = 'local';
+  const brokerId = 1;
+
+  const renderComponent = () =>
+    render(
+      <WithRoute path={clusterBrokerPath(':clusterName', ':brokerId')}>
```

Comment:
for the query params we should not send any other parameter since its default should be the params , to avoid repetition and thus not having any different params.